### ISSUE TITLE
Playlist menu is not showing playlists that I have stored from others

### ISSUE THREAD

**Issue by jcjdammers** [anxdpanic/plugin.video.youtube#1023]
Hi,

When I select the Playlists menu in the addon, it is only showing the private playlists that I have created myself in YouTube. It is not showing the public playlists (from others) that I have stored/added in my YouTube library.

I am using version v7.1.1.6 of the addon.
I am running on LibreELEC 12.0.1 with Kodi (Omega) v21.1 (for Generic PC)

Thanks,
Jeroen.

**Comment by MoojMidge** [comment]
This functionality is not currently supported.

As a workaround you can search for the playlists in the addon and then add them to your bookmarks via the context menu.

**Comment by jcjdammers** [comment]
@MoojMidge ,
Thanks for the workaround. Hope you can add this functionality in a next release.

**Comment by MoojMidge** [comment]
It is a longer term goal to add this and other related features, but no estimate for when.

**Comment by MoojMidge** [comment]
[v7.3.0+alpha.1](https://github.com/anxdpanic/plugin.video.youtube/releases/tag/v7.3.0%2Balpha.1) has been released with support added for showing saved playlists.

**Comment by demouser007** [comment]
Should saved playlists be working now? v7.3.0+beta.3 and v7.3.zip do not display them.

[kodi.log](https://github.com/user-attachments/files/22396147/kodi.log)

**Comment by MoojMidge** [comment]
Yes, but you are opening your own playlists.

Saved playlists are shown separately and can enabled from `Settings > Folders > Saved Playlists`

**Comment by demouser007** [comment]
Yes, I just opened "playlists", since there are no Saved Playlists in the Folders section of the settings. I double-checked with a clean installation of plugins on a clean installation of Kodi (on Windows). In versions v7.3.0+alpha.1, v7.3.0+beta.3, and v7.3.zip, the Saved Playlists folder is not displayed. As far as I can see, the log does not show which folders are available in the settings.

**Comment by MoojMidge** [comment]
Yeah it would help if it wasn't invisible. Will fix it up in the next release. In the meantime you can just delete the following line:

https://github.com/anxdpanic/plugin.video.youtube/blob/2853f3b03df39d5208915257ad29308d1a02a605/resources/settings.xml#L515

So this:
```
                <setting id="youtube.folder.saved.playlists.show" type="boolean" label="30611" help="">
                    <level>0</level>
                    <default>false</default>
                    <visible>false</visible>
                    <control type="toggle"/>
                </setting>
```

Becomes:
```
                <setting id="youtube.folder.saved.playlists.show" type="boolean" label="30611" help="">
                    <level>0</level>
                    <default>false</default>
                    <control type="toggle"/>
                </setting>
```
